corehq/apps/sso/models.py

```diff
@@ -128,11 +128,6 @@ def get_sso_exempt_users(self):
             email_domain__identity_provider=self,
         ).values_list('username', flat=True)
 
-    @classmethod
-    def domain_has_identity_provider(cls, domain):
-        owner = BillingAccount.get_account_by_domain(domain)
-        return cls.objects.filter(owner=owner, is_editable=True).exists()
-
```

Comment:
Interesting -- I probably would have left this as is. I've been trying to move any `objects` access to within the associated model class, to make mocking and reading easier. What pushed you to move this to its own module?

Reply:
i guess renaming it would make more sense...hmm. on second thought I'm going to move this back

Reply:
what it really needs is a rename